.github/workflows/ci.yml

```
@@ -0,0 +1,855 @@
+      COMMIT_SHA: ${{ github.sha }}
+      REPOSITORY: ${{ github.repository }}
+    steps:
+      - name: No-op on successful pipelines
+        if: ${{ success() }}
+        run: echo "CI run succeeded; notification dispatch skipped."
+
+      - name: Dispatch Slack alert (if configured)
+        if: ${{ failure() && env.CI_FAILURE_WEBHOOK_URL != '' }}
+        env:
+          WEBHOOK_URL: ${{ secrets.CI_FAILURE_WEBHOOK_URL }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:54:8: Double quote to prevent globbing and word splitting

(shellcheck)